address-space-controller/image-template.yaml

```diff
@@ -50,5 +51,5 @@ run:
           - "/opt/address-space-controller.jar"
 osbs:
       repository:
-            name: containers/amqmaas-1
-            branch: rh-amqmaas-1.0-address-space-controller-openshift-rhel-7
+            name: containers/amq-online-1
+            branch: amq7-amq-online-1-address-space-controller-openshift-container
```

Comment:
@lulf the branch will need to match what is being entered into comet and should have rhel-7